node_cli/realm/realmAuth.js

```diff
@@ -0,0 +1,39 @@
+const Realm = require("realm");
+const index = require("../index");
+const config = require("../config");
+
+const REALM_APP_ID = config.realmAppId;
+const appConfig = {
+  id: REALM_APP_ID,
+  timeout: 10000,
+};
+
+const app = new Realm.App(appConfig);
+
+async function login(email, password) {
```

Comment:
If it's app.logIn(), this should be logIn() too for your text's sake.

Reply:
Not sure what you mean by "for your text's sake", and not sure what the reason is they should match.

Reply:
Consistency. Your text is going to look like this:
>To log in, call the login function, which will call the logIn method() on the Realm...